file/writable_file_writer.cc

```diff
@@ -470,6 +470,11 @@ IOStatus WritableFileWriter::WriteBuffered(const char* data, size_t size) {
       allowed = left;
     }
 
+#ifndef NDEBUG
+    size_t* call_back_para[] = {&allowed, &size};
+#endif
+    TEST_SYNC_POINT_CALLBACK("SplitWrite:allowed", call_back_para);
```

Comment:
Should we need a test sync point, it's recommended to name it `WritableFileWriter::WriteBuffered:AfterComputingAllowedSize`.

Reply:
fix it, remove the split write operation, only use concurrent write to reproduce this issue.